src/test/java/net/masterthought/cucumber/ReportGenerator.java

```diff
@@ -62,7 +60,6 @@ public ReportGenerator() {
     }
 
     protected void setUpWithJson(String... jsonFiles) {
-        Logger.getLogger(ReportParser.class.getName()).setLevel(Level.WARNING);
```

Comment:
the reason of this was to switch off logging mess that was produced every time tests were running - as long as tests pass such logging were not helpful at all

Reply:
With logic in this PR, it will not spam the output with logging and given logging adjustments its faster in general.  Overall though should not really notice much when it comes to the build.